epicsdbbuilder/recordbase.py

```diff
@@ -158,6 +163,14 @@ def Print(self, output):
             print('    field(%s, %s%s)' % (k, padding, value), file = output)
         for alias in sorted(list(self.__aliases)):
             print('    alias("%s")' % alias, file = output)
+        for name, info in list(self.__infos):
+            if isinstance(info, str):
+                # A string value, double quote it
+                info = '"%s"' % info
+            else:
+                # A JSON structure, don't quote it
+                info = '\n    '.join(json.dumps(info, indent=4).splitlines())
```

Comment:
Hmm.  Is `json.dumps` going to do all the necessary db safe quoting for us?  Otherwise we could have some nasty surprises here...

Reply:
Looks like it does